kview/src/main/resources/dev/ikm/komet/kview/controls/concept-navigator.css

```diff
@@ -112,7 +115,7 @@
 }
 
 .tree-cell {
-    -fx-background: -Grey-3;
+    -fx-background: transparent;
```

Comment:
Can we make this style more specific, instead of just applying to all ".tree-cell"? By more specific I mean adding the parents, etc so that there's little probably of this style clashing with other styles defined in Komet.

Reply:
done